charts/helm-broker/templates/deploy.yaml

```diff
@@ -138,8 +138,8 @@ spec:
             command:
               - curl
               - localhost:{{ .Values.ctrl.statusPort }}/live
-          periodSeconds: 10
-          timeoutSeconds: 2
+          periodSeconds: 300
```

Comment:
liveness with 5minutes periods? maybe start with 1minute, processing such event should not kill us. IMO 5min is definitely too much